src/client/components/CvPdfRender.js

```diff
@@ -0,0 +1,23 @@
+import React,{useState,useEffect} from 'react';
```

Comment:
@KavyaReddy732 do you have formatting on? If you do, you can make it automatically happen when saving a file. Here is how: try it out and see if it changes anything: https://stackoverflow.com/questions/39494277/how-do-you-format-code-on-save-in-vs-code

Reply:
i am lost with the styling of iframe. i tried to style but it's not making any changes